docs/TUTORIAL.md

```diff
@@ -41,160 +47,307 @@ This tutorial uses the following billable components of Google Cloud:
+For this reference guide, you need a Google Cloud [project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects). 
+You can create a new one, or select a project you already created:
 
 1. Select or create a Google Cloud project.
     [GO TO THE PROJECT SELECTOR PAGE](https://console.cloud.google.com/projectselector2/home/dashboard)
 
 2. Enable billing for your project.
     [ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
 
-3. If you will be using a Cloud Shell instance as your development environment, then in the Cloud Console, activate Cloud Shell.
+3. If you will be using a Cloud Shell instance as your development environment, then in the Cloud Console, activate 
+   Cloud Shell.
     [ACTIVATE CLOUD SHELL](https://console.cloud.google.com/?cloudshell=true&_ga=2.175791653.601714487.1648649282-1707141935.1648504534)
 
-    At the bottom of the Cloud Console, a Cloud Shell session starts and displays a command-line prompt. Cloud Shell is a shell environment with the Google Cloud CLI already installed and with values already set for your current project. It can take a few seconds for the session to initialize.
+    At the bottom of the Cloud Console, a Cloud Shell session starts and displays a command-line prompt. Cloud Shell 
+is a shell environment with the Google Cloud CLI already installed and with values already set for your current project. 
+It can take a few seconds for the session to initialize.
 
-When you finish this tutorial, you can avoid continued billing by deleting the resources you created. See [Clean up](#clean-up) for more detail.
+When you finish this tutorial, you can avoid continued billing by deleting the resources you created. 
+See [Clean up](#clean-up) for more detail.
 
 ---
 ## Prepare your environment
-1. Clone the gitHub repository to your development environment
-1. Install  dbt:
+1. In Cloud Shell, specify the project that you want to use for this tutorial:
+   ```
+   gcloud config set project PROJECT_ID
+   ```
+   Replace PROJECT_ID with the ID of the project that you selected or created for this tutorial. 
+
+   If a dialog displays, click *Authorize*.
+
+
+2. Specify a default region to use for infrastructure creation:
+   ```
+   gcloud config set compute/region REGION
+   ```
+
+3. Create and activate a Python virtual environment:
+   ```
+   python -m venv reg-rpt-env
+   source reg-rpt-env/bin/activate
+   ```
+
+   You see that your command-line prompt is prefixed with the name of the virtual environment.
+
+
+4. Clone the gitHub repository to your development environment
+   ```
+   git clone "https://github.com/GoogleCloudPlatform/reg-reporting-blueprint"
+   ```
+
+
+5. Install Terraform. To learn how to do this installation, see the [HashiCorp documentation](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform).
```

Comment:
Terraform for Cloud shell already seems installed, e.g.,
https://github.com/hashicorp/terraform-getting-started-gcp-cloud-shell/blob/master/tutorial/cloudshell_tutorial.md

Reply:
I added that for consistency with our blog article here - https://cloud.google.com/architecture/set-up-regulatory-reporting-architecture-bigquery